ArWikiCats/new_resolvers/relations_resolver/countries_names_double_v2.py

```
@@ -0,0 +1,228 @@
+
+    result = double_bot_v1().search_all_category(category)
+    logger.info_if_or_debug(f"<<yellow>> end resolve_v1: {category=}, {result=}", result)
+    return result
+
+
+@functools.lru_cache(maxsize=10000)
+def resolve_v2(category: str) -> str:
+    """
+    Examples:
+        >>> # Example of a dual-country resolution (v2)
+        >>> resolve_v2("jordan–iraq relations")
```

Ruff (0.14.11)

Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)